### ISSUE TITLE
[8.3.0] Ponds on mountains (Taiga / Ice / Whatever) look ugly and behave oddly

### ISSUE THREAD

**Issue by D3nnis3n** [StrangeLoopGames/EcoIssues#13859]
![world](https://user-images.githubusercontent.com/25908592/69445759-463beb00-0d53-11ea-831b-8771758dbc44.png)

**Comment by elfl0rd** [comment]
At the same time, many of them look beautiful

**Comment by D3nnis3n** [comment]
But some are broken :D

**Comment by elfl0rd** [comment]
Look this not broken :D
![20191123224001_1](https://user-images.githubusercontent.com/4980243/69484335-2126a580-0e43-11ea-9aff-8e075c1393cf.jpg)
![20191123223832_1](https://user-images.githubusercontent.com/4980243/69484336-21bf3c00-0e43-11ea-9245-e42b398bb08f.jpg)
![20191123223853_1](https://user-images.githubusercontent.com/4980243/69484337-21bf3c00-0e43-11ea-8af4-22c51368ff5f.jpg)
![20191123223906_1](https://user-images.githubusercontent.com/4980243/69484338-2257d280-0e43-11ea-9edb-41a40d95cfff.jpg)
![20191123223933_1](https://user-images.githubusercontent.com/4980243/69484339-2257d280-0e43-11ea-99b9-be4e6b9c8e4e.jpg)

**Comment by D3nnis3n** [comment]
I had lots of the broken ones in the last cycles. Didn't say there aren't any working ones :)

**Comment by elfl0rd** [comment]
I just don't want this problem to be solved by removing the lakes from these biomes :))
Ofc, if fix possible - it will be great.

**Comment by D3nnis3n** [comment]
That's not my intention either.

**Comment by redwyre** [comment]
Do all the words that have had this problem have modifications to the world gen?

**Comment by D3nnis3n** [comment]
Yes, but no modifications regarding to lakes and just because i never play with default worlds.
They all have exactly two modifications:

Size from 72x72 to 160x160 and Land to Water from 45 55 to 75 25.
